.travis.yml

```diff
@@ -5,7 +5,9 @@ php:
   - 5.4
   - 5.5
   - 5.6
-  - 7
+  - 7.0
+  - 7.1
+  - nightly
```

Comment:
Sorry for the delay.  I updated the changes to allow `nightly` failures.  I have not had time to fix the test suite for 7.1 though.